app/assets/stylesheets/charts.scss

```diff
@@ -62,6 +62,12 @@
   margin: 0px auto 27px auto;
 }
 
+.statistics-pie-placeholder{
+  margin: 12px auto;
+  width: 240px;
+  height: 268px;
+}
+
 .statistics-pie{
```

Comment:
instead of duplicating the css above, you can change this to `.statistics-pie, .statistics-pie-placeholder{` and add the `small` class to the placeholder div.